{"failedTests":[{"testName":"parallel/test-without-async-context-frame","timestamp":1737747913416,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 0,win10-COMPILED_BY-vs2022 #32261","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win10-COMPILED_BY-vs2022/32261/","buildNumber":32261,"builtOn":"test-azure_msft-win10_vs2019-x64-1","nodeVersion":"24.0.0","commitHash":"1120f7f8b083f33df527d990c104442e91d0a8e5","callStack":[{"upstreamBuild":68072,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":77281,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":64714,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"56747"}],"tap":{"ok":false,"name":"parallel/test-without-async-context-frame","id":995,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"▶ without AsyncContextFrame\n  ✖ async-hooks\\test-async-local-storage-args.js (2577.8809ms)\n    AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-args.js failed with exit code 3221225477\n    \n    3221225477 !== 0\n    \n        at TestContext.<anonymous> (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-without-async-context-frame.mjs:59:7)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async Test.run (node:internal/test_runner/test:981:9)\n        at async Promise.all (index 0)\n        at async Suite.run (node:internal/test_runner/test:1360:7)\n        at async startSubtestAfterBootstrap (node:internal/test_runner/harness:297:3) {\n      generatedMessage: false,\n      code: 'ERR_ASSERTION',\n      actual: 3221225477,\n      expected: 0,\n      operator: 'strictEqual'\n    }\n\n  ✔ async-hooks\\test-async-local-storage-async-await.js (1815.7378ms)\n  ✔ async-hooks\\test-async-local-storage-async-functions.js (1253.3958ms)\n  ✔ async-hooks\\test-async-local-storage-dgram.js (1108.8149ms)\n  ✔ async-hooks\\test-async-local-storage-enable-disable.js (1168.429ms)\n  ✔ async-hooks\\test-async-local-storage-enter-with.js (1093.353ms)\n  ✔ async-hooks\\test-async-local-storage-errors.js (963.174ms)\n  ✔ async-hooks\\test-async-local-storage-gcable.js (1051.8765ms)\n  ✔ async-hooks\\test-async-local-storage-http-agent.js (1042.0166ms)\n  ✔ async-hooks\\test-async-local-storage-http.js (999.0992ms)\n  ✔ async-hooks\\test-async-local-storage-misc-stores.js (957.696ms)\n  ✔ async-hooks\\test-async-local-storage-nested.js (934.5294ms)\n  ✔ async-hooks\\test-async-local-storage-no-mix-contexts.js (1300.2339ms)\n  ✔ async-hooks\\test-async-local-storage-promises.js (999.0283ms)\n  ✔ async-hooks\\test-async-local-storage-socket.js (1001.2249ms)\n  ✔ async-hooks\\test-async-local-storage-thenable.js (1002.712ms)\n  ✔ async-hooks\\test-async-local-storage-tlssocket.js (1111.3204ms)\n  ✔ parallel\\test-async-local-storage-bind.js (982.261ms)\n  ✔ parallel\\test-async-local-storage-contexts.js (1001.0631ms)\n  ✔ parallel\\test-async-local-storage-deep-stack.js (990.4446ms)\n  ✔ parallel\\test-async-local-storage-exit-does-not-leak.js (1063.4675ms)\n  ✔ parallel\\test-async-local-storage-http-multiclients.js (1194.6077ms)\n  ✔ parallel\\test-async-local-storage-snapshot.js (1056.5259ms)\n✖ without AsyncContextFrame (26680.239ms)\nℹ tests 23\nℹ suites 1\nℹ pass 22\nℹ fail 1\nℹ cancelled 0\nℹ skipped 0\nℹ todo 0\nℹ duration_ms 26689.7985\n\n✖ failing tests:\n\ntest at test\\parallel\\test-without-async-context-frame.mjs:49:5\n✖ async-hooks\\test-async-local-storage-args.js (2577.8809ms)\n  AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-args.js failed with exit code 3221225477\n  \n  3221225477 !== 0\n  \n      at TestContext.<anonymous> (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-without-async-context-frame.mjs:59:7)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async Test.run (node:internal/test_runner/test:981:9)\n      at async Promise.all (index 0)\n      at async Suite.run (node:internal/test_runner/test:1360:7)\n      at async startSubtestAfterBootstrap (node:internal/test_runner/harness:297:3) {\n    generatedMessage: false,\n    code: 'ERR_ASSERTION',\n    actual: 3221225477,\n    expected: 0,\n    operator: 'strictEqual'\n  }"},"time":27154.372,"fullname":"parallel/test-without-async-context-frame","closingTestPoint":false}}]}